### DISCUSSION TITLE
Drawer only icon mode

### DISCUSSION THREAD

**Discussion by AfganAbbas** [saadeghi/daisyui#1693] [Ideas / feature requests]
![image](https://user-images.githubusercontent.com/89972922/225332832-63e618f9-42f7-457b-acd0-89b540da4e9e.png)
It would be good if daisyUI have this kind of drawer. If is it possible to customize available drawers to this, please help me about it

**Comment by saadeghi** [comment]
The `drawer` itself doesn't have a width. if you look at the examples, there's `w-80` class. you can use another size and the menu will be smaller. For example: https://play.tailwindcss.com/yHoR8qLfVw?size=1068x720

If you want to give the user an option to collapse/uncollapse the menu (icon+text or icon only), you can use JS to add remove class names (`w-80` or `w-20`) and also hide the text part of the links (because in smaller menu, only icon should be visible)

Let me know if you have any questions.

**Reply by AfganAbbas** [reply]
Thanks for answer, it is pretty clear, I have one more question. I want to make sidebar, like sidebar of this website(https://www.crazygames.com/). In your opinion, is it possible to convert DaisyUI sidebar into this?
